drafts/use-cases/4.obtaining-single-event.md

```diff
@@ -22,13 +22,14 @@ It wouldn't be possible for the calendar to show all of the event's attributes,
 thus there is a need to show those outstanding ones in a separate view (i.e. popup, new screen, etc.).
 The application sends a GET request to the selected event's resource identifier:
 
-```
+```http
 GET /api/events/1
 
 HTTP 200 OK
```

Comment:
Using two blocks makes it appear as the HTTP 200 OK is still part of the request. I'd say we use 1 or 3 (request headers, response headers, payload)